docs/advanced_usage.rst

```diff
@@ -34,6 +34,12 @@ hooks, as these can be run on any platform. However, if you intend for your
 template to only be run on a single platform, a shell script (or `.bat` file
 on Windows) can be a quicker alternative.
 
+.. note::
+    Make sure your hook scripts work in a robust manner.  If a hook script
+    fails (that is, if it finishes with an exit status that is not one of
+    success), the project generation will stop and the generated directory will
```

Comment:
You could refer to the [official sys.exit docs](https://docs.python.org/2/library/sys.html#sys.exit) to be very explicit.

>  If it is an integer, zero is considered “successful termination” and any nonzero value is considered “abnormal termination” by shells and the like.